invenio_app_ils/records/loaders/schemas/json/document_request.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""DocumentRequest schema for marshmallow loader."""
+
+from invenio_records_rest.schemas.fields import PersistentIdentifier
+from marshmallow import Schema, fields
+
+
+class DocumentRequestSchemaV1(Schema):
+    """Document Request schema."""
+
+    pid = PersistentIdentifier()
+    state = fields.Str(required=True)
+    patron_pid = fields.Str(required=True)
```

Comment:
for the `patron_pid`, I think we have to create a specific validator to validate that if it is the user creating the request, it must be him, pseudo-code:

```
    @validates('patron_pid')
    def validate_patron_pid(self, value):
        if not current_user == admin or librarian:
            return patron_pid ==  current_user
```